js/views/pauseScreen.js

```diff
@@ -45,7 +45,7 @@ class PauseScreen extends React.Component {
     this.handleResize();
     this.hideVrPauseButton();
     document.addEventListener('mousemove', this.handlePlayerMouseMove, false);
-    document.addEventListener('touchmove', this.handlePlayerMouseMove, false);
```

Comment:
Will this affect other mobile/tablet environments such as Android or the Microsoft Surface?

Reply:
@aeng7 touch event is supported for these [browsers](https://caniuse.com/#feat=touch)
passiveSupported works for these [browsers](https://caniuse.com/#feat=passive-event-listener) . As I see all supported touch event browsers support passiveSupported.

But you right that this changes will affect mobile vr video behavior. Because of that I asked QA Engineers to do regression after merge.

Reply:
Sounds good, thanks for the update!